bloom-housing/bloom
# feat: required labels on listings management fields

# Pull Request Template

## Issue

Addresses #1580


## Description

Add required notes to the required fields in listings management. Had to add a few optional props to the field components.

## Type of change


## How Can This Be Tested/Reviewed?

Open listings management and ensure before you've submitted that it looks good and then try to publish without entering anything and ensure the notes still look good when you also have error messages.

## Checklist:
